PixelHunter1995/GameStates/Playing.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using System;
+using System.Collections.Generic;
+
+namespace PixelHunter1995.GameStates
+{
+    /// <summary>
+    /// State when playing. Will probably be replaced with many different states.
+    /// </summary>
+    class Playing : IGameState
+    {
+        private readonly Texture2D background;
+        private readonly Texture2D guy;
+        private readonly SpriteBatch spriteBatch;
+
+        public Playing(Texture2D background, Texture2D guy, SpriteBatch spriteBatch)
+        {
+            this.background = background;
+            this.guy = guy;
+            this.spriteBatch = spriteBatch;
+        }
+
+        public void Draw(GameTime gameTime)
+        {
+            spriteBatch.Begin();
+            spriteBatch.Draw(background, Vector2.Zero, Color.White);
+            spriteBatch.Draw(guy, new Vector2(20,20), Color.White);
+            spriteBatch.End();
```

Comment:
I think that maybe these `spriteBatch.Begin()`/`spriteBatch.End()` should be around all draw function calls?